packages/Path-View-Core.package/SDiagramMorphToolbar.class/instance/addButton.model.action.enabled.edge.tooltip..st

```diff
@@ -0,0 +1,15 @@
+adding
+addButton: aLabelMorph model: anObject action: aSelector enabled: aSymbol edge: anEdge tooltip: aTooltipString
```

Comment:
existed before, we added tooltip param. this is a factory mehod for, guess... button morphs. there is a function #tooltipButtons which returns an array of configurations which items then are passed to this method